requirements-UBUNTU.txt

```diff
@@ -1,2 +1,3 @@
 ethtool
 ipaddr
+distro
```

Comment:
could you do the same you did for wok and create a requirements.txt to be used by CI ?